src/Commands/RunRepeater.ts

```diff
@@ -115,6 +115,7 @@ export class RunRepeater implements CommandModule {
         }
       })
       .option('experimental-connection-reuse', {
+        alias: 'ntlm',
```

Comment:
Please add the deprecation notice:
```suggestion
        alias: 'ntlm',
        deprecate: 'Use --ntlm instead',
```
To be more consistent and avoid ambiguity, you can also introduce a new option `--ntlm` instead of the alias, adding the deprecated notice to the existing one.

Reply:
@mongoose79 not resolved. 
> you can also introduce a new option --ntlm instead of the alias

right now with ntlm option alias there will be deprecation warning